mod/wikirate_source/set/type/source/source.rb

```diff
@@ -131,7 +131,7 @@ def file_link? url
   # prevent from showing file too big while users are adding a link source
   max_size = (max = Card['*upload max']) ? max.db_content.to_i : 5
 
-  !(content_type.start_with?("text/html") || 
+  !(content_type.start_with?("text/html") ||
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.